src/unit_tests/wazuh_db/test_wdb_global_parser.c

```diff
@@ -0,0 +1,187 @@
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "wazuh_db/wdb.h"
+
+int __wrap__mdebug1()
+{
+    return 0;
+}
+
+int __wrap__mdebug2()
+{
+    return 0;
+}
+
+int __wrap__mwarn()
+{
+    return 0;
+}
+
+int __wrap__merror()
+{
+    return 0;
+}
+
+int __wrap_wdb_open_global2()
+{
+    return mock();
+}
+
+void __wrap_wdb_leave(){}
+
+int __wrap_wdb_exec()
+{
+    return mock();
+
+}
+
+typedef struct test_struct {
+    wdb_t *socket;
+    char *output;
+} test_struct_t;
+
+static int test_setup(void **state) {
+    test_struct_t *init_data;
+    init_data = malloc(sizeof(test_struct_t));
```

Comment:
Probably should be better to use `os_calloc` here. This ensures that the memory apart from being allocated, it is initialized to zero.